Inc/TASRun.h

```diff
@@ -65,6 +65,11 @@ typedef struct
 	Transition transitions_dpcm[MAX_TRANSITIONS];
 	uint8_t console_data_size;
 	uint8_t input_data_size;
+	uint32_t moder_firstLatch;
+
+	uint8_t multitap;
+	int32_t blank;
+	char inputFile[256];

```

Comment:
Do we want to support wchar / unicode?

Reply:
The fonts in the SSD1306 OLED display library (and all the other ones I looked at) only have support for basic latin characters anyway (program memory is limited afterall), so I don't see the benefit in supporting unicode. It will still be able to open files with characters beyond 0x7F just the filename won't display properly.